packages/compass-components/src/components/leafy-green/combobox/util.tsx

```diff
@@ -0,0 +1,121 @@
+import { isComponentType, keyMap as _keyMap } from '@leafygreen-ui/lib';
+import { kebabCase } from 'lodash';
+import React from 'react';
+import type { ComboboxOptionProps } from './combobox-types';
+
+// TODO - remove this when lib/keyMap supports Backspace & Delete
+export const keyMap = {
+  ..._keyMap,
+  Backspace: 8,
+  Delete: 46,
+} as const;
+
+const escapeRegExp = (input: string) => {
+  return input.replace(/[-[/\]{}()*+?.,\\^$|#\s]/g, '\\$&');
+};
+
+/**
+ *
+ * Wraps the first instance of `wrap` found in `str` within the provided `element`.
+ *
+ * E.g. `wrapJSX('Apple', 'ap', 'em') => <><em>Ap</em>ple</>`
+ *
+ * @param str
+ * @param wrap
+ * @param element
+ * @returns `JSX.Element`
+ */
+export const wrapJSX = (
+  str: string,
+  wrap?: string,
+  element?: string
+): JSX.Element => {
+  if (wrap && element) {
+    const regex = new RegExp(escapeRegExp(wrap), 'gi');
```

Comment:
Should we add a comment here to mark that we changed something from the original code?